Classes/Command/InstallCommandController.php

```diff
@@ -41,24 +42,24 @@ class InstallCommandController extends CommandController
     protected $packageStatesGenerator;
 
     /**
-     * TYPO3 Setup
+     * TYPO3 Setup.
      *
      * Use as command line replacement for the web installation process.
      * Manually enter details on the command line or non interactive for automated setups.
      *
-     * @param bool $nonInteractive If specified, optional arguments are not requested, but default values are assumed.
-     * @param bool $force Force installation of TYPO3, even if <code>LocalConfiguration.php</code> file already exists.
-     * @param string $databaseUserName User name for database server
+     * @param bool   $nonInteractive       If specified, optional arguments are not requested, but default values are assumed.
```

Comment:
Hm, this might affect the documentation and help texts as well. Besides that, all other classes are formatted with a single space to separate the argument types, names and description.

I have no plans to change that